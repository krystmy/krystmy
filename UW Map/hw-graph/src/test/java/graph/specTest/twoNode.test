# Test adding two node to a graph

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

# Prints out nodes in graph
ListNodes graph1