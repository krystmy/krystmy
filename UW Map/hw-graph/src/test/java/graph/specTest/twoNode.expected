# Test adding two node to a graph

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1
added node n2 to graph1

# Prints out nodes in graph
graph1 contains: n1 n2